data-sets-tests/src/test/java/org/zowe/data/sets/tests/DataSetsGetIntegrationTest.java

```diff
@@ -32,27 +30,10 @@ public void testGetValidDataset() throws Exception {
         createDataSet(pdsRequest);
 
         try {
+            DataSet expected = DataSet.builder().name(tempDataSet).migrated(false).build();
```

Comment:
Please revert this change and add a new test for your endpoint rather than delete the old one.